src/pages/mobile/TransactionDetails.vue

```diff
@@ -51,6 +51,12 @@
       :fee="transaction.tx.fee"
     />
 
+    <DetailsAddress
+      v-if="transaction.tx.accountId"
+      :name="$t('transfer.transaction.details.account')"
+      :address="transaction.tx.accountId"
+    />
+
```

Comment:
Transaction details header not very informative. My proposal is to have at least transaction type in header in such cases.
![image](https://user-images.githubusercontent.com/9008074/64218572-90a48d00-cf05-11e9-8347-d8af223dc206.png)